### DISCUSSION TITLE
SP111 v1.1 plug not enough memory from tasmota-minimal.bin to tasmota.bin v10

### DISCUSSION THREAD

**Discussion by DariBer** [arendst/Tasmota#13853] [Q&A] [answered]
I have some SP111 v1.1 wall plugs that I have been working fine and I have been able to upgrade with out any problems. But now I have problem to upgrade to version 10. I upgraded first to the minimal of v10 and then I tried to upgrade to the "normal" tasiota.bin. but I get error message "Upload Failed, Not enough space"

**Comment by sfromis** [comment, marked as answer]
For ESP8266 based devices, always, always use the compressed `.bin.gz` binaries, only ever use any `.bin` for first Tasmota binary flashed or with "too old" Tasmota versions.

**Reply by DariBer** [reply]
Yes that did it.

Sorry I didn't know boy that I should use the .bin.gz file. :(

Thank you so much for your help
